app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -30,9 +30,22 @@ interface FormattedLocation {
   weekdayHours: string;
 }
 
+const getCsrfToken = () => {
+  const meta: HTMLMetaElement | null = document.querySelector('meta[name="csrf-token"]');
+
+  return meta?.content;
+};
```

Comment:
Generic parameter of `querySelector` could simplify this a bit.

```suggestion
const getCsrfToken = () =>
  document.querySelector<HTMLMetaElement>('meta[name="csrf-token"]')?.content;
```